docs/tutorials/qGAN.ipynb

```diff
@@ -0,0 +1,2644 @@
+{
```

Comment:
<p>I need to add an explaination of how this all works. However, it currently doesn't work, so I cannot write it yet</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/quantum/pull/339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>